LICENSE

```diff
@@ -106,7 +106,7 @@ users' freedom, it does ensure that the user of a program that is
 linked with the Library has the freedom and the wherewithal to run
 that program using a modified version of the Library.
 
-  The precise terms and conditions for copying, distribution and
+  The precise terms and conditions for copying, distribution, and
```

Comment:
The usage of the sequence comma is not mandatory, so unless there's an ambiguity (and in this case there isn't), both ways are valid.